server/db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE videos_recommendation;
+
+CREATE TABLE videos (
+    id VARCHAR(255) PRIMARY KEY,
```

Comment:
We typically use `SERIAL` as a type of ids, rather than `VARCHAR`s - can you research `SERIAL` and work out what it does and why it's useful?